cicd/gitops/gitops-release-notes.adoc

```diff
@@ -0,0 +1,21 @@
+//OpenShift Pipelines Release Notes
+include::modules/gitops-document-attributes.adoc[]
+[id="gitops-release-notes"]
+= {gitops-title} release notes
+:context: gitops-release-notes
+include::modules/common-attributes.adoc[]
```

Comment:
You will need to add an include statement for modules/gitops-document-attributes.adoc too